packages/eas-cli/src/build/utils/resourceClass.ts

```diff
@@ -8,24 +8,30 @@ import { BuildResourceClass } from '../../graphql/generated';
 import Log from '../../log';
 import { getReactNativeVersionAsync } from '../metadata';
 
+type AndroidResourceClass = Exclude<
+  ResourceClass,
+  | ResourceClass.M1_EXPERIMENTAL
+  | ResourceClass.M1_MEDIUM
+  | ResourceClass.M1_LARGE
+  | ResourceClass.INTEL_MEDIUM
+  | ResourceClass.M_MEDIUM
+  | ResourceClass.M_LARGE
+>;
+
 const iosResourceClassToBuildResourceClassMapping: Record<ResourceClass, BuildResourceClass> = {
   [ResourceClass.DEFAULT]: BuildResourceClass.IosDefault,
   [ResourceClass.LARGE]: BuildResourceClass.IosLarge,
-  [ResourceClass.M1_EXPERIMENTAL]: BuildResourceClass.IosM1Large,
-  [ResourceClass.M1_MEDIUM]: BuildResourceClass.IosM1Medium,
+  [ResourceClass.M1_EXPERIMENTAL]: BuildResourceClass.IosMMedium,
+  [ResourceClass.M1_MEDIUM]: BuildResourceClass.IosMMedium,
   [ResourceClass.M1_LARGE]: BuildResourceClass.IosM1Large,
```

Comment:
```suggestion
  [ResourceClass.M1_LARGE]: BuildResourceClass.IosMLarge,
```
Shouldn't this also be changed?